<launch>
    <!-- Arguments -->
    <arg name="x_pos" default="-1.1"/>
    <arg name="y_pos" default="4.2"/>
    <arg name="z_pos" default="0.0"/>
    <arg name="a_theta" default="0.0"/>

    <node name="main" pkg="minitask5" type="main.py" output="screen">
        <param name="max_green_boxes"             value="2"/>
        <param name="max_red_hydrants"            value="2"/>
    </node>
    <!-- <node name="goal_finder" pkg="minitask5" type="goal_finder.py" output="screen"/> -->
    <!-- <node name="image_processor" pkg="minitask5" type="image_proc.py" output="screen"/> -->
    <!-- <node name="object_avoid" pkg="minitask5" type="object_avoid.py" output="screen"/> -->
    <!-- <node name="search" pkg="minitask5" type="search.py" output="screen"/> -->
    <!-- <node name="localisation" pkg="minitask5" type="localisation.py" output="screen"/> -->
    <!-- <node name="mapping" pkg="minitask5" type="mapping.py" output="screen"/> -->

    <!-- Navigation -->
    <include file="$(find minitask5)/launch/minitask5_navigation.launch">
        <!-- <arg name="x_nav_pos"   value="-1.3"/> -->
    </include>
</launch>
